src/Domain/DonationService.php

```diff
@@ -164,6 +164,8 @@ function () use ($donation) {
                 'amount' => $donation->getAmountFractionalIncTip(),
                 'currency' => strtolower($donation->getCurrencyCode()),
                 'description' => $donation->getDescription(),
+                'capture_method' => 'automatic', // 'automatic' was default in previous API versions,
+                                                // default is now 'automatic_async'
```

Comment:
Yep, so far seems to make sense to avoid automatic_async. https://docs.stripe.com/payments/payment-intents/asynchronous-capture

It sounds like in return for a modest speed up in a call that is already quite fast, async mode would require us to start doing checks later in the process, e.g. perhaps making sure donors cannot press the final donate button too early if fees haven't yet been calculated. And opening up the possibility of `charge.succeeded` not always having properties we need to check, meaning we have to start thinking about later charge events too, seems like a very significant downside.